index.d.ts

```diff
@@ -1,53 +1,87 @@
-export type StringComparator = (left: string, right: string) => number;
-
-/**
-Ascending sort comparator.
-
-@example
-```
-import alphaSort = require('alpha-sort');
-
-['b', 'a', 'c'].sort(alphaSort.ascending);
-//=> ['a', 'b', 'c']
-```
-*/
-export const ascending: StringComparator;
-
-/**
-Descending sort comparator.
-
-@example
-```
-import alphaSort = require('alpha-sort');
-
-['b', 'a', 'c'].sort(alphaSort.descending);
-//=> ['c', 'b', 'a']
-```
-*/
-export const descending: StringComparator;
-
-/**
-Case-insensitive ascending sort comparator.
-
-@example
-```
-import alphaSort = require('alpha-sort');
-
-['B', 'a', 'C'].sort(alphaSort.caseInsensitiveAscending);
-//=> ['a', 'B', 'C']
-```
-*/
-export const caseInsensitiveAscending: StringComparator;
-
-/**
-Case-insensitive descending sort comparator.
-
-@example
-```
-import alphaSort = require('alpha-sort');
-
-['B', 'a', 'C'].sort(alphaSort.caseInsensitiveDescending);
-//=> ['C', 'B', 'a']
-```
-*/
-export const caseInsensitiveDescending: StringComparator;
+declare namespace alphaSort {
+	type StringComparator = (left: string, right: string) => number;
+
+	type Options = {
+		/**
+		Whether or not to sort in descending order.
+
+		@default false
+		*/
+		readonly descending?: boolean;
+
+		/**
+		Whether or not to sort case-insensitively.
+
+		Note: If two elements are considered equal in the case-insensitive comparison, the tie-break will be a standard (case-sensitive) comparison.
+
+		@example
+		```
+		const alphaSort = require('alpha-sort');
+
+		['bar', 'baz', 'Baz'].sort(alphaSort({caseInsensitive: true}));
+		//=> ['bar', 'Baz', 'baz']
+		```
+
+		@default false
+		*/
+		readonly caseInsensitive?: boolean;
+
+		/**
+		Whether or not to sort using natural sort order (such as sorting `10` after `2`).
+
+		Note: If two elements are considered equal in the natural sort order comparison, the tie-break will be a standard (non-natural) comparison.
+
+		@default false
+		*/
+		readonly natural?: boolean;
```

Comment:
Can you add an example here too? And in the readme.

Reply:
Done 
c425b8a